src/Transpiler.ts

```diff
@@ -1269,8 +1269,9 @@ export abstract class LuaTranspiler {
                 params = this.transpileArguments(node.arguments, sig);
                 return `(rawget(${expr}, ${params} )~=nil)`;
             } else {
-                const type = this.checker.getTypeAtLocation(node.expression);
-                const op = tsHelper.isFunctionWithContext(type, this.checker) ? ":" : ".";
+                const sigDecl = sig.getDeclaration();
+                const op = !sigDecl || tsHelper.getDeclarationContextType(sigDecl, this.checker) !== ContextType.Void
```

Comment:
Try not to abbreviate variables too much, I don't think there is a lot of abbreviations on other variable names, we should probably try to be somewhat consistent.